src/main/java/it/gov/pagopa/payhub/activities/activity/ingestionflow/receipt/PaymentsReportingImplicitReceiptHandlerActivityImpl.java

```diff
@@ -0,0 +1,56 @@
+package it.gov.pagopa.payhub.activities.activity.ingestionflow.receipt;
+
+import it.gov.pagopa.payhub.activities.connector.classification.PaymentsReportingService;
+import it.gov.pagopa.payhub.activities.connector.debtposition.ReceiptService;
+import it.gov.pagopa.payhub.activities.connector.organization.OrganizationService;
+import it.gov.pagopa.payhub.activities.dto.classifications.PaymentsReportingTransferDTO;
+import it.gov.pagopa.payhub.activities.exception.InvalidValueException;
+import it.gov.pagopa.payhub.activities.mapper.ingestionflow.receipt.PaymentsReporting2ReceiptMapper;
+import it.gov.pagopa.pu.classification.dto.generated.PaymentsReporting;
+import it.gov.pagopa.pu.debtposition.dto.generated.ReceiptDTO;
+import it.gov.pagopa.pu.debtposition.dto.generated.ReceiptWithAdditionalNodeDataDTO;
+import it.gov.pagopa.pu.organization.dto.generated.Organization;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Lazy
+@Slf4j
+@Component
+public class PaymentsReportingImplicitReceiptHandlerActivityImpl implements PaymentsReportingImplicitReceiptHandlerActivity {
+	private static final List<String> PAYMENT_OUTCOME_CODES = List.of("8", "9");
+
+	private final PaymentsReportingService paymentsReportingService;
+	private final OrganizationService organizationService;
+	private final PaymentsReporting2ReceiptMapper paymentsReporting2ReceiptMapper;
+	private final ReceiptService receiptService;
+
+	public PaymentsReportingImplicitReceiptHandlerActivityImpl(PaymentsReportingService paymentsReportingService,
+	                                                           OrganizationService organizationService,
+	                                                           PaymentsReporting2ReceiptMapper paymentsReporting2ReceiptMapper,
+	                                                           ReceiptService receiptService) {
+		this.paymentsReportingService = paymentsReportingService;
+		this.organizationService = organizationService;
+		this.paymentsReporting2ReceiptMapper = paymentsReporting2ReceiptMapper;
+		this.receiptService = receiptService;
+	}
+
+	@Override
+	public void handle(PaymentsReportingTransferDTO paymentsReportingTransferDTO) {
+		if (PAYMENT_OUTCOME_CODES.contains(paymentsReportingTransferDTO.getPaymentOutcomeCode())) {
```

Comment:
log as INFO on the else in order to inform that it has been invoked even if not required

Reply:
ps: log the the entire DTO, it doesn't contain PII, right?